console/src/layout/slice.ts

```diff
@@ -249,7 +249,10 @@ export const { actions, reducer } = createSlice({
     remove: (state, { payload: { keys } }: PayloadAction<RemovePayload>) => {
       keys.forEach((contentKey) => {
         const layout = select(state, contentKey);
-        if (layout == null) return;
+        if (layout == null || layout.key == MAIN_WINDOW) {
+          console.log("MAIN WINDOW - NOT CLOSING SLICE");
```

Comment:
Doesn't really make sense to log this message when `layout == null`

Reply:
Removed